test1.html

```diff
@@ -151,11 +151,11 @@ <h3>SOCIAL CONNECTION</h3>
         <li class="info-item"><img src="#" alt="G+">GOOGLE+</li>
         <li class="info-item"><img src="#" alt="youtube">YOUTUBE</li>
       </ul>
-    </section>
-    <section class="footer-copyright">
+    </article>
+    <article class="footer-copyright">
       <p>&copy;2018-COPYRIGHT OF SPECIALIZED CLINC</p>
       <p>DESIGNoBY<a href="#">DD.COM</a></p>  
```

Comment:
Should not type upper-case this text.
Using CSS for transform the text to upper-case
What is `Designoby` mean?